scripts/zkai_d128_recursive_pcd_route_selector_gate.py

```
@@ -0,0 +1,1023 @@
+        require_str(route.get("claim_boundary"), f"{route_id}.claim_boundary", layer="route_table")
+        require_str(route.get("next_action"), f"{route_id}.next_action", layer="route_table")
+        require_str(route.get("blocking_missing_object"), f"{route_id}.blocking_missing_object", layer="route_table")
+        require_object(route.get("evidence"), f"{route_id}.evidence", layer="route_table")
+
+    expect_equal(
+        by_id["local_stwo_nested_verifier_air"]["status"],
+        "NO_GO_MISSING_NESTED_VERIFIER_AIR",
+        "local nested verifier status",
+        layer="route_table",
+    )
+    expect_equal(by_id["local_stwo_nested_verifier_air"]["usable_today"], False, "local nested verifier usable", layer="route_table")
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)